apps/src/netsim/NetSimTable.js

```diff
@@ -133,11 +172,11 @@ NetSimTable.prototype.readAll = function (callback) {
  */
 NetSimTable.prototype.makeThrottledRefresh_ = function (waitMs) {
   return _.throttle(function () {
-    this.clientApi_.all(function (err, data) {
-      if (err === null) {
-        this.fullCacheUpdate_(data);
-      }
-    }.bind(this));
+    if (this.useIncrementalRefresh_) {
```

Comment:
So, entirely collapse `refreshAll` and `refreshFromRowID` into a `refresh` function that behaves differently depending on the table configuration?